spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/annotation/ContinueSpan.java

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
I'd leave them in Sleuth so that we see that they are of any value. If more people will use it we can consider pushing it somewhere else. WDYT?